crates/bevy_pbr/src/atmosphere/resources.rs

```diff
@@ -186,6 +188,8 @@ impl FromWorld for RenderSkyBindGroupLayouts {
                     (4, uniform_buffer::<GpuLights>(true)),
                     (5, texture_2d(TextureSampleType::Float { filterable: true })), //transmittance lut and sampler
                     (6, sampler(SamplerBindingType::Filtering)),
+                    (7, texture_2d(TextureSampleType::Float { filterable: true })), //multiscattering lut and sampler
+                    (8, sampler(SamplerBindingType::Filtering)),
```

Comment:
just leaving a note that most of these bindgroups can now use sequential. Not necessary to switch them if we're gonna keep iterating on this and potentially changing them back to not be sequential. If we do switch to sequential it should be in a different PR though.

Reply:
I'll have a cleanup PR for the atmosphere bindings when I work on the shadows, because I want to somehow reuse the shadow sampling functions between PBR and atmosphere shaders. I'll need to consult with you and others about this